File: app/src/main/java/com/karim/booksapp/data/models/pojos/ImageLinks.java

package com.karim.booksapp.data.models.pojos;

import com.squareup.moshi.Json;

public class ImageLinks {

    @Json(name = "smallThumbnail")
    public String smallThumbnail;
    @Json(name = "thumbnail")
    public String thumbnail;

}

File: app/src/main/java/com/karim/booksapp/data/models/pojos/ImageLinks.kt
package com.karim.booksapp.data.models.pojos

import com.squareup.moshi.Json

class ImageLinks {

    @Json(name = "smallThumbnail")
    var smallThumbnail: String? = null
    @Json(name = "thumbnail")
    var thumbnail: String? = null

}
